RELEASING.md

```diff
@@ -76,9 +88,11 @@ the failures have been resolved.
    (the `git log` is going to show you things in submodules, which are not going
    to be part of your release).
 1. Edit `CHANGES.md` to include a summary of the changes.
-1. In `internal/version/version.go`, update `const Repo` to today's date with the format `YYYYMMDD`.
+1. In `internal/version/version.go`, update `const Repo` to today's date with
```

Comment:
Do we have a tracking bug for this?

Reply:
Created: https://github.com/googleapis/google-cloud-go/issues/3109